app/javascript/data/upsells.ts

```diff
@@ -131,7 +131,7 @@ export const pauseUpsell = async (id: string) => {
   const response = await request({
     method: "POST",
     accept: "json",
-    url: Routes.checkout_upsell_pause_url(id),
+    url: Routes.checkout_upsell_pause_path(id),
```

Comment:
Used `_path` instead of `_url` for relative API endpoints. Fixes tests failing due to hostname mismatches.